src/renderers/react/useCustomFieldViewState.ts

```diff
@@ -1,30 +1,24 @@
-import type { MutableRefObject } from "react";
-import { useEffect, useRef, useState } from "react";
+import { useEffect, useState } from "react";
 import { CustomFieldView } from "../../plugin/fieldViews/CustomFieldView";
 import type { CustomField } from "../../plugin/types/Element";
 
 export const useCustomFieldViewState = <Data extends unknown>({
```

Comment:
Probably should have got this in the types PR but maybe we could rename it to `useCustomFieldViewState` -> `useCustomFieldState`?

Reply:
👍